build_overrides/v8.gni

```diff
@@ -12,7 +12,10 @@ enable_correct_v8_arch = false
 # TODO: Set v8_use_external_startup_data to true on Windows and iOS to match
 # GYP.
 # iOS - http://crbug.com/545676
-v8_use_external_startup_data = !is_ios
+declare_args() {
```

Comment:
This could be replaced by a backport of [this upstream commit](https://crrev.com/0fffeb2adaa3c284b760922c1aecce1516b998ce).